src/main.cpp

```diff
@@ -7,8 +7,8 @@ int main() {
     print(numbers);
     sort(numbers);
     print(numbers);
-    auto minimum = min(numbers);
```

Comment:
I think that according to task description we should remain everything from master branch and merge additional reverse function

Reply:
thanks for advice!